problems/fibonacci/Fibonacci.java

```diff
@@ -0,0 +1,46 @@
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+import org.junit.Assert;
+
+public class Fibonacci {
+	
+	
+	public int FibonacciaAt(int number) {
+		List<Integer> fibonacciaSequence = new ArrayList<Integer>();
+		fibonacciaSequence.add(0);
+		fibonacciaSequence.add(1);
+		
+		if (number > 2) {
+			while (number > fibonacciaSequence.size()) {
+				int nextNumberInSequence = fibonacciaSequence.get(fibonacciaSequence.size()-1) + fibonacciaSequence.get(fibonacciaSequence.size()-2);
+				fibonacciaSequence.add(nextNumberInSequence);
+			}
+			
+		} else {
+			return fibonacciaSequence.get(number-1);
+		}
+		
+		return fibonacciaSequence.get(number-1);
```

Comment:
You should simplify your if-statement. You could (should) completely get rid of the "else" case. 
```java
fibonacciaSequence.get(number-1);
```
is executed in any case, so removing it makes the code easier to understand overall.